lib/index.js

```diff
@@ -531,9 +584,10 @@ FSTree.prototype._track = function(operation, entry) {
       entry
     ]) - 1;
   } else {
-    // TODO: if entry is the same, but different operation, throw error?
+    if (this._changes[position][0] !== operation) {
+      throw new Error('Two different operation is being done on the same file.');
```

Comment:
@stefanpenner you had originally made this case explicitly update the operation but it seems to me that if we do two different ops on the same file in a single build this should be an error (or at least a warning).  In any case if it's somehow valid we can't mutate in place due to ordering.

Was there some case / plugin you had in mind that would do two different operations on the same output file within a single build?